flocker/node/agents/test/test_cinder.py

```diff
@@ -61,3 +67,65 @@ def test_verify_false_ca_path(self):
             'verify_ca_path': '/a/path'
         }
         self.assertEqual(_openstack_verify_from_config(**config), False)
+
+
+class FakeCinderClient(object):
+    """"
+    Fake implementation of the cinder volume manager to use in the Cinder
+    Destroy tests.
+    Right now, we don't need a full mock of the class, we just need the
+    get not to raise any exception so the destroy volume timeout test
+    actually times out - it will be trying to get the volume in a loop
+    after deleting it, expecting a ``CinderNotFound`` exception that will
+    never happen, so we can verify that the timeout exception is rised
+    (see FLOC-1853)
+    """
+    def delete(self, volume_id):
+        """
+        A no-op delete.
+        We do not need the delete to do anything for the timeout tests.
```

Comment:
Even stronger than this, we need for deletion to not succeed (at least not quickly) for the timeout tests.